convert_sdf_to_usd/tutorial.md

```diff
@@ -0,0 +1,44 @@
+# Convert SDF to USD
+
+## Prerequisites
+
+* sdformat with USD support (see the [sdformat installation instructions](/tutorials?tut=install&ver=>%3D+12))
```

Comment:
can you remove `&ver=>%3D+12` from this link? the link appears broken for me like that. Tested from:

* http://sdformat.org/tutorials?tut=convert_sdf_to_usd&cat=developers&branch=usd-convert